sdk/eventhub/Azure.Messaging.EventHubs/TROUBLESHOOTING.md

```diff
@@ -356,6 +356,8 @@ When filing GitHub issues, the following details are requested for all scenarios
   - 88 (Buffered Producer Publish Start)
   - 89 (Buffered Producer Publish Complete)
   - 90 (Buffered Producer Publish Error)
+  - 124 (Buffered Producer Idle Start)
+  - 125 (Buffered PRoducer Idle Complete)
```

Comment:
```suggestion
  - 125 (Buffered Producer Idle Complete)
```

Reply:
Wow... I should really avoid throwing out a PR at the end of the day!